src/pages/trading-specification/_use-trading-spec-columns.tsx

```diff
@@ -0,0 +1,84 @@
+import { createColumnHelper } from '@tanstack/react-table'
+import React, { useMemo } from 'react'
+import { THeaders } from './_types'
+import { TableCell, TableHeaderCell } from './components/_elements'
+import { Localize } from 'components/localization'
+
+const liveMarketColumnHelper = createColumnHelper<THeaders>()
+
+const useLiveColumns = () => {
+    const columns = useMemo(() => {
+        return [
+            liveMarketColumnHelper.accessor('instrument', {
+                header: () => (
+                    <TableHeaderCell text={<Localize translate_text="Instrument name" />} />
```

Comment:
For all the Localization let's use _t_STRING_t_

Reply:
We have information. regarding this in wiki: https://wikijs.deriv.cloud/en/Frontend/deriv-static/07-translation-workflow/Translation-Key-Extraction-POC#best-practices